src/sentry/api/base.py

```diff
@@ -205,6 +206,23 @@ def build_cursor_link(self, request: Request, name: str, cursor: Cursor):
     def convert_args(self, request: Request, *args, **kwargs):
         return (args, kwargs)
 
+    def permission_denied(self, request, message=None, code=None):
+        """
+        Raise a specific superuser exception if the user can become superuser
+        and the only permission class is SuperuserPermission. Otherwise, raises
+        the appropriate exception according to parent DRF function.
+        """
+        permissions = self.get_permissions()
+
+        can_be_superuser = request.user.is_authenticated and request.user.is_superuser
```

Comment:
this is 👍  don't need to check for active superuser because we already check it in the permission class